code-examples/core_features/getting_started/3_getting_started.ts

```diff
@@ -0,0 +1,5 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export async function main() {
+  await Kilt.ChainHelpers.BlockchainApiConnection.getConnectionOrConnect()
```

Comment:
Isn't there also a simple connect method?

```suggestion
  await Kilt.connect()
```

Reply:
I think this is a better method for the beginning. Since you may have an open connection, allow them to connect to it. 

However, if they didn't close a connection, you could potentially have multiple connections.

Reply:
[It's the same function](https://github.com/KILTprotocol/sdk-js/blob/9e132611abc0dffbb22062f655e7bef3ce39cbc0/packages/core/src/kilt/Kilt.ts#L29)

Reply:
Updated.